src/main/java/com/uid2/admin/Main.java

```diff
@@ -291,8 +292,9 @@ public void run() {
                 }
             }
 
+            boolean bThrowOnS3WriteException = config.getBoolean(throwOnS3WriteException, true);
```

Comment:
We don't use Hungarian notation in the current conventions. If the constant is renamed to all caps, we can just use the camel case version.